GW2EIEvtcParser/EIData/ProfHelpers/ProfHelper.cs

```diff
@@ -13,12 +13,14 @@ internal static class ProfHelper
 
         private static readonly List<InstantCastFinder> _genericInstantCastFinders = new List<InstantCastFinder>()
         {
-            new DamageCastFinder(SigilOfEarth, SigilOfEarth).UsingICD(500), // Earth Sigil
-            new DamageCastFinder(LightningStrikeSigil, LightningStrikeSigil).UsingICD(500), // Air Sigil
-            new DamageCastFinder(FlameBlastSigil, FlameBlastSigil).UsingICD(500), // Fire Sigil
-            new DamageCastFinder(SigilOfHydromancy, SigilOfHydromancy).UsingICD(500), // Hydro Sigil
-            new EXTHealingCastFinder(WaterBlastCombo1, WaterBlastCombo1), // Water Blast Combo
+            new DamageCastFinder(SigilOfEarth, SigilOfEarth).UsingICD(500),
+            new DamageCastFinder(LightningStrikeSigil, LightningStrikeSigil).UsingICD(500),
+            new DamageCastFinder(FlameBlastSigil, FlameBlastSigil).UsingICD(500),
+            new DamageCastFinder(SigilOfHydromancy, SigilOfHydromancy).UsingICD(500),
+            new EXTHealingCastFinder(WaterBlastCombo1, WaterBlastCombo1),
             new EffectCastFinderByDst(RuneOfNightmare, EffectGUIDs.RuneOfNightmare),
+            new BuffGainCastFinder(PortalEntranceWhiteMantleWatchwork, PortalWeavingWhiteMantleWatchwork),
+            new BuffGainCastFinder(PortalExitWhiteMantleWatchwork, PortalUsesWhiteMantleWatchwork).UsingBeforeWeaponSwap(true),
```

Comment:
Why UsingBeforeWeaponSwap?

Reply:
Using the skill makes you drop the bundle and in my test log the portal exit skill ended up after the weapon swap (bundle drop). I think it makes sense to place it before the swap.